src/nspawn/nspawn.c

```diff
@@ -2721,6 +2743,11 @@ static int mount_tunnel_dig(const char *root) {
         const char *p, *q;
         int r;
 
+        if (!arg_privileged) {
+                log_debug("Not digging mount tunnel, because running unprivileged.");
```

Comment:
These days can't we use the new mount api instead of the mount tunnel stuff? Via the new component?

Reply:
We certainly wn't need th mount tunnel anymore with the new mount API. But so far i haven't implemented bind mounting during runtime for unpriv containers and it's not clear to me that we can even do that without any privs. It's something to look into in the future.

Reply:
will add to TODO file